packages/dashmate/src/listr/tasks/buildServicesTaskFactory.js

```diff
@@ -21,10 +21,12 @@ function buildServicesTaskFactory(
 
         const obs = await dockerCompose.build(envs);
 
-        await new Promise((res, rej) => {
-          obs
-            .subscribe(task.stdout().write, rej, res);
-        });
+        if (ctx.isVerbose) {
+          await new Promise((res, rej) => {
```

Comment:
I believe we need to wait for a promise anyways but subscribe to stdout only in case of verbose 1

Reply:
I don't understand how observers work, I can't figure out where to put the conditional logic for that.